docs/development/Serial.md

```diff
@@ -2,19 +2,19 @@
 
 Betaflight has enhanced serial port flexibility but configuration is slightly more complex as a result.
 
-Betaflight has the concept of a function (MSP, GPS, Serial RX, etc) and a port (VCP, UARTx, SoftSerial x).
-Not all functions can be used on all ports due to hardware pin mapping, conflicting features, hardware, and software
-constraints.
+Betaflight has the concept of a function (MSP, GPS, Serial RX, etc) and a port (VCP, UARTx, SoftSerial x, LPUART1).
+Not all functions can be used on all ports due to hardware pin mapping, conflicting features, hardware, and software constraints.
 
 ## Serial Port Types
 
 - USB Virtual Com Port (VCP) - USB pins on a USB port connected directly to the processor without requiring
   a dedicated USB to UART adapter. VCP does not 'use' a physical UART port.
 - UART - A pair of dedicated hardware transmit and receive pins with signal detection and generation done in hardware.
 - SoftSerial - A pair of hardware transmit and receive pins with signal detection and generation done in software.
+- LPUART - A "Low Power" UART format available on G4 and other MCU's. By default LPUARTs are limited to 9600 Baud, but Betaflight reconfigures them to work just like a normal UART. Typically there is only one LPUART, LPUART1. It's pin assignment is normally shown as `SERIAL_TX 11` and `SERIAL_RX 11` with the `RESURCES` command.
```

Comment:
```suggestion
- LPUART - A "Low Power" UART format available on G4 and other MCU's. By default LPUARTs are limited to 9600 Baud, but Betaflight reconfigures them to work just like a normal UART. Typically there is only one LPUART, LPUART1. It's pin assignment is normally shown as `SERIAL_TX 11` and `SERIAL_RX 11` with the `RESOURCE` command.
```

Reply:
Thanks very much for proof-reading so carefully